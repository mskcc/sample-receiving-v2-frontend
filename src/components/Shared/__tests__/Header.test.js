import React, { Component } from 'react'
import renderer from 'react-test-renderer'
import TestHeader from '../Header'
import { BrowserRouter as Router, Route } from 'react-router-dom'

const Header = props => (
  <Router>
    <TestHeader />
  </Router>
)

describe('Render Header', () => {
  it('render Header correctly', () => {
    const HeaderComponent = renderer.create(<Header />).toJSON()
    expect(HeaderComponent).toMatchSnapshot()
  })
})
