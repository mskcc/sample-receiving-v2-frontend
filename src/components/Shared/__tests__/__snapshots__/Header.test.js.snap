// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Header render Header correctly 1`] = `
<header
  className="MuiPaper-root-14 MuiPaper-elevation4-20 MuiAppBar-root-5 MuiAppBar-positionStatic-9 MuiAppBar-colorPrimary-12 Header-header-1"
  title="msk.png"
>
  <div
    className="MuiToolbar-root-41 MuiToolbar-regular-43 MuiToolbar-gutters-42"
  >
    <div
      className="MuiAvatar-root-45"
    >
      <img
        alt="mskcc logo"
        className="MuiAvatar-img-47"
        src="msk.png"
      />
    </div>
    <h6
      className="MuiTypography-root-48 MuiTypography-h6-65 MuiTypography-colorInherit-77 Header-title-2"
    >
      SampleReceiving V2
    </h6>
    <button
      className="MuiButtonBase-root-110 MuiButton-root-84 MuiButton-text-86 MuiButton-flat-89"
      disabled={false}
      onBlur={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="0"
      type="button"
    >
      <span
        className="MuiButton-label-85"
      >
        <a
          aria-current={null}
          className="Header-navlink-3"
          href="/upload"
          onClick={[Function]}
        >
          <h6
            className="MuiTypography-root-48 MuiTypography-h6-65 MuiTypography-colorInherit-77"
          >
            Upload
          </h6>
        </a>
      </span>
      <span
        className="MuiTouchRipple-root-113"
      />
    </button>
    <button
      className="MuiButtonBase-root-110 MuiButton-root-84 MuiButton-text-86 MuiButton-flat-89"
      disabled={false}
      onBlur={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="0"
      type="button"
    >
      <span
        className="MuiButton-label-85"
      >
        <a
          aria-current={null}
          className="Header-navlink-3"
          href="/promote"
          onClick={[Function]}
        >
          <h6
            className="MuiTypography-root-48 MuiTypography-h6-65 MuiTypography-colorInherit-77"
          >
            Promote
          </h6>
        </a>
      </span>
      <span
        className="MuiTouchRipple-root-113"
      />
    </button>
  </div>
</header>
`;
